### DISCUSSION TITLE
Typos

### DISCUSSION THREAD

**Discussion by deregtervictor** [OSTA-group/osta#64] [General]
I came across some things that could be written more clearly.

1. "ADD A NEW AREA" could be -> "ADD NEW AREA"
![Screenshot 2024-06-04 at 09-48-30 Ionic App](https://github.com/OSTA-group/osta/assets/101172722/d02bc8ee-3fb9-44a7-9201-d62757a47ca4)

2. Both are true but use the same text for the button and header
![Screenshot 2024-06-04 at 09-49-08 Ionic App](https://github.com/OSTA-group/osta/assets/101172722/088955b4-d628-44c0-81a0-61c1faa86200)

Feel free to discuss more typos in this discussion.

**Comment by BauwenDR** [comment]
Since you are downloading data places with extensions, I think it might be best to have both buttons state "DOWNLOAD" and add "NEW" on the areas overview screen

**Reply by BauwenDR** [reply]
This inconsistency has been fixed. Feel free to use this discussion for other types in the app, documentation or elsewhere in our projects
